src/prefect/tasks/zendesk/zendesk_tasks.py

```diff
@@ -0,0 +1,211 @@
+from collections import defaultdict
+from datetime import datetime
+import os
+
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+from prefect.engine.signals import FAIL
+
+import requests
+import time
+from typing import List, Union
+
+
+class ZendeskTicketsIncrementalExportTask(Task):
+    """
+    This task can be used to perform an incremental export of tickets from Zendesk.
+    More info at
+    https://developer.zendesk.com/api-reference/ticketing/ticket-management/incremental_exports/#incremental-ticket-export
+
+    Args:
+        - subdomain (str, optional): The Zendesk subdomain to use to export tickets.
+        - email_address (str, optional): The email address to use to authenticate on Zendesk
+        - api_token (str, optional): The API token to use to athenticate on Zendesk
+            If passed, it will take precedence over `api_token_env_var`.
+        - api_token_env_var (str, optional): The name of the env var which contains the
+            API token to use to authenticate on Zendesk.
+        - start_time (int, datetime, optional): The start time to use to export tickets.
+            Can be passed as an epoch timestamp or a `datetime` object.
+        - cursor (str, optional): The cursor to use to export tickets.
+            If passed, it will take precedence over `start_time`.
+        - exclude_deleted: (bool, optional): Whether to exclude deleted tickets or not.
```

Comment:
What is the default?

Reply:
I described the default value at line 32